packages/SketchMorph2-Core.package/MorphicMonet.class/instance/initialize.st

```diff
@@ -4,7 +4,7 @@ initialize
 	self 
 		initializePanels
 		setLabel: 'Morphic Monet';
-		addMorph: self canvasPanel frame: ((0@self defaultTopPanelHeight) corner: 1 asPoint);
-		addMorph: self topPanel frame: (0 asPoint corner: (1@self defaultTopPanelHeight));
+		addMorph: self canvasPanel frame: ((0@self class topPanelHeight) corner: 1 asPoint);
+		addMorph: self topPanel frame: (0 asPoint corner: (1@self class topPanelHeight));
```

Comment:
When we already have to edit this methods, we could take the opportunity to add spaces around the `@`:
```suggestion
		addMorph: self canvasPanel frame: ((0 @ self class topPanelHeight) corner: 1 asPoint);
		addMorph: self topPanel frame: (0 asPoint corner: (1 @ self class topPanelHeight));
```